e2e/integration/apicatalogtests.js

```diff
@@ -0,0 +1,45 @@
+context( 'Home Page Tests', () => {
+    before( () => {
+        cy.visit( Cypress.env( 'QA_HOST' ) +'developers/api-catalog');
+        cy.get( '#username', { timeout: 5000 } ).type( Cypress.env( 'USERNAME' ) );
+        cy.get( '#password' ).type( Cypress.env( 'PASSWORD' ) );
+        cy.get( '#submit' ).click();
+    } );
+    Cypress.on( 'uncaught:exception', ( err, runnable ) => {
+        return false;
+    } );
+    it( 'Add API Test', () => {
+        cy.contains( 'Add API' ).click( { force: true } );
+    } )
+    it( 'Fill the add API form', () => {
+        cy.faker = require( 'faker' );
+        const words = cy.faker.lorem.words();
+        cy.get( 'input[name="name"]' ).type( words );
+        cy.get( 'textarea[name="description"]' ).type( "test" )
+        cy.contains( "REST" ).click( { force: true } )
+        cy.get( 'div[class="pf-c-select__toggle-wrapper"] > input[placeholder^="Search"]' ).type( 'one-portal' ).then( () => {
+            cy.wait(5000)
+            cy.contains( 'one-portal-test', { timeout: 10000 } ).should( 'be.visible' ).click( { force: true } )
+        })
+        cy.get( '[name="appUrl"]' ).type( "https://www.google.com" )
+        cy.get( '[name="schemaEndpoint"]' ).type( "https://www.redhat.com" )
```

Comment:
```suggestion
        cy.get( '[name="schemaEndpoint"]' ).type( "https://petstore.swagger.io/v2/swagger.json" )
```

Reply:
Hi @riginoommen , I have updated this in https://github.com/1-Platform/one-platform/pull/1317/commits/5c5599e2d31a8fab11c05cb778b1fd2c725c6261